addons/l10n_vn/migrations/17.0.2.0.2/post-migration.py

```diff
@@ -12,7 +12,7 @@ def _fix_accounts_type(env):
         domains_per_company = []
         for company in env['res.company'].with_context(active_test=False).search([('chart_template', '=', 'vn')]):
             doamin = expression.AND([
-                [('company_id', '=', company.id), ('account_type', '!=', correct_account_type)],
+                [('company_ids', '=', company.id), ('account_type', '!=', correct_account_type)],
```

Comment:
Using an array operator seems more logical with an m2m field
```suggestion
                [('company_ids', 'in', [company.id]), ('account_type', '!=', correct_account_type)],
```